src/components/List/ListItem/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Pressable, StyleProp, Text, View, ViewStyle } from 'react-native';
+import useStyles from './styles';
+import { ItemHeight } from '../../../utils/itemSize';
+import Checkbox from '../../Checkbox';
+import Icon from '../../Icon';
+
+type ContentFunc = ({ disabled }: { disabled?: boolean; checked?: boolean }) => React.ReactNode;
+
+export interface IBaseListItem {
+  title: string;
+  subtitle?: string;
+  label?: string;
+  leftContent?: React.ReactNode | ContentFunc;
+  rightContent?: React.ReactNode | ContentFunc;
+  onPress?: () => void;
+  testID?: string;
+  style?: StyleProp<ViewStyle>;
+  checked?: boolean;
+  itemHeight: ItemHeight;
+  selection?: 'none' | 'check-icon' | 'check-box';
+  disabled?: boolean;
+}
+
+const ListItem: React.FC<IBaseListItem> = ({
+  label,
+  title,
+  subtitle,
+  rightContent,
+  leftContent,
+  onPress,
+  checked,
+  itemHeight,
+  selection,
+  disabled,
+  ...rest
+}) => {
+  const styles = useStyles(itemHeight, !!disabled);
+  const [isChecked, setChecked] = useState(checked);
```

Comment:
should be setIsChecked to follow the common practice

Reply:
FIxed